users/kimurayoshimi/9.js

```diff
@@ -0,0 +1,8 @@
+const a = (b) => b.includes("e");
+const b = a(["a", "b", "c"]);
+console.log(b);
+
+const c = (d) => d.find((value) => value === "e");
```

Comment:
これは実行していないですが必要ですか？

Reply:
`a()` was being executed.😢
And with the fixed code, `undefined` was returned. I was able to confirm that this is the case because the method looks for `e`.